Apps/WebClient/src/ClientApp/app/components/navmenu/navHeader.vue

```diff
@@ -49,33 +43,22 @@
           id="menuBtndUser"
           :text="greeting"
           right
+          variant="dark"
         >
           <b-dropdown-item>
-            <router-link id="menuBtnLogout" to="/logout">
+             <router-link  variant="primary"  to="/timeLine">
+                <span  class="fa fa-stream" ></span> Timeline
+             </router-link>
+             <br></br>
```

Comment:
I dont think this is valid inside the list. Is giving me compilation errors

Reply:
I think it's the <br> I  used I wasn't sure how to create a break like a list.